sentry/src/main/java/io/sentry/event/interfaces/SentryStackTraceElement.java

```diff
@@ -116,9 +116,24 @@ public String getPlatform() {
     public static SentryStackTraceElement[] fromStackTraceElements(StackTraceElement[] stackTraceElements,
                                                                    Frame[] cachedFrames) {
         SentryStackTraceElement[] sentryStackTraceElements = new SentryStackTraceElement[stackTraceElements.length];
-        for (int i = 0; i < stackTraceElements.length; i++) {
-            sentryStackTraceElements[i] = fromStackTraceElement(stackTraceElements[i],
-                cachedFrames != null ? cachedFrames[i].getLocals() : null);
+        for (int i = 0, j = 0; i < stackTraceElements.length; i++, j++) {
```

Comment:
Comment here explaining the rationale behind this new implementation would be helpful for future readers — it makes sense in context but I think without the knowledge of this change it'd be pretty arcane